### ISSUE TITLE
Reduce overhead when submitting small amounts of data to KCIDB

### ISSUE THREAD

**Issue by gctucker** [kernelci/kcidb#88]
Submitting test results from one job currently takes about 1s.  For example, from the kernelci-backend Celery log:
```
[ INFO/MainThread] Task kcidb-tests[44910df2-393d-457d-8c56-198d2bfcf54c] succeeded in 1.19923461694s: ObjectId('5ecfb49e5841ebffd3c9164f')
```

These tasks all appear to last between 0.8s and 1.4s.  That seems like quite a long time given the small amount of data involved in each set of results, and it doesn't vary with the size of the data. Some test jobs only have 5 test case results and take the same amount of time.

While it's possible to do things in kernelci-backend, such as buffering the data to submit less often or keep a process alive with a connection always open, it seems like any new submitter would face the same problem.  So it would be beneficial to have it solved once in KCIDB rather than in each system that submits some data.

**Comment by spbnick** [comment]
Support for #92 is merged now, but #119 would improve things further.

**Comment by spbnick** [comment]
#115 is fixed now, which further improves performance of submitting many small reports to kcidb. Closing until we're dissatisfied again :grin:
